lib/interceptor.js

```diff
@@ -8,11 +8,20 @@ var interceptor = {
     }
     done(window[NAMESPACE]);
   },
+
+  excludeUrls: function excludeUrls(urls, done) {
+    var NAMESPACE = '__webdriverajax';
```

Comment:
```suggestion
    const NAMESPACE = '__webdriverajax';
```

Reply:
`const` is not used in the browser library currently, so this would be a larger change at odds with the rest of the code.

(IE won't choke on parsing it, but it doesn't support the full featureset either, which is why it isn't used.)